Dockerfile

```diff
@@ -8,6 +8,9 @@ RUN apt-get update -y && \
 
 RUN apt install -y curl wget mc gcc make
 
+RUN apt-get update -y && \
+    apt install -y chromium
```

Comment:
As long as chrome driver and related libraries are really dependent on versions used - please put here not any, but the specific version